gateway-ha/src/main/java/io/trino/gateway/ha/config/RulesExternalConfiguration.java

```diff
@@ -18,7 +18,7 @@
 public class RulesExternalConfiguration
 {
     private String urlPath;
-    private List<String> blackListHeaders;
+    private List<String> excludeHeaderList;
```

Comment:
How about renaming to `excludeHeaders` (removing List)?

Reply:
good idea

Reply:
yup changed it. 
![image](https://github.com/user-attachments/assets/a358a93f-b2c8-405a-b430-42ee00783c61)
![image](https://github.com/user-attachments/assets/d858b280-98a0-42b2-8b29-d2e1bf7afa07)